service/internal/auth/config.go

```diff
@@ -4,8 +4,9 @@ import "fmt"
 
 // AuthConfig pulls AuthN and AuthZ together
 type Config struct {
-	Enabled     bool `yaml:"enabled" default:"true" `
-	AuthNConfig `mapstructure:",squash"`
+	Enabled      bool     `yaml:"enabled" default:"true" `
+	PublicRoutes []string `mapstructure:"-"`
```

Comment:
What does the `"-"` mean?

Reply:
basically ignores the field when encoding/decoding the struct.

Reply:
Isn't the point that this can be extended? Or does this force it to be configured in code?

Reply:
Forces it to be configured through code. I felt like it could potentially open up issues if we allowed it to be configured via the config file.